node-js-basics-101/node_modules/ws/index.js

```diff
@@ -0,0 +1,15 @@
+/*!
+ * ws: a node.js websocket client
+ * Copyright(c) 2011 Einar Otto Stangvik <einaros@gmail.com>
+ * MIT Licensed
+ */
+
+'use strict';
+
+const WebSocket = require('./lib/WebSocket');
```

Comment:
__[jshint]__
* 'const' is available in ES6 (use 'esversion: 6') or Mozilla JS extensions (use moz).
* Redefinition of 'WebSocket'.
* 'require' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->